### ISSUE TITLE
Warning useNativeDriver from RadioButton

### ISSUE THREAD

**Issue by hugosbg** [callstack/react-native-paper#1870]
## Environment
react: 16.11.0
react-native: 0.62.2
react-native-paper: ^3.8.0

## Description

when I check an option   

![image](https://user-images.githubusercontent.com/14058763/80547083-afbcc700-898d-11ea-98ed-d46deac4d0b9.png)

## Reproducible Demo
```
<RadioButton
     value='male'
     status={getValues('male') === 'male' ? 'checked' : 'unchecked'}
     onPress={() => setValue('gender', 'male')}
/>
<RadioButton
     value='female'
     status={getValues('female') === 'female' ? 'checked' : 'unchecked'}
     onPress={() => setValue('gender', 'female')}
/>
```

**Comment by CarpeNecopinum** [comment]
Checkbox elements show the same issue.

**Comment by Trancever** [comment]
Do you want to submit a PR with a fix? The only thing to do is to pass `useNativeDriver: false` to animation in RadioButton.Android component.

**Comment by CarpeNecopinum** [comment]
PRed a fix. Apparently for the Checkbox component this was fixed with 3.9.0 already. So this one only addresses the RadioButton component.
